bricepollock/canyoneer
# [ISSUE-37] Add go-to on map for canyons found in favorites and add coordinate information / copy

### Description
Add button to move the map to a canyon from CanyonView page (only from favorites)
* Add coordinates section to view coordinate of canyon
* Add copy button next to coordinates that copies coordinates to pasteboard and presents toast to users

### Test Plan
* Tap into a canyon from map (no 'go to on map' button)
* See location section
* Tap copy button by coordinate
* "Copied" toast shows up
* Toast dismisses on its own
* Paste coordinates into another mapping app
* Return to Canyoneer
* Favorite this canyon
* Navigate map somewhere else
* Go to favorites
* Tap on that same map
* See "go to on map" button visible
* Tap "Go to on map" button
* App switches tap to "Map"
* Map now centered on that favorited canyon

(Important: make sure to test launching from favorite'd canyon before going to the map. It tests the map initialization race with updating coordinate)
